.gitignore

```diff
@@ -2,6 +2,7 @@
 /vendor
 /.vendor-new
 .vscode
+/.idea
```

Comment:
Do you mind adding this in your personal `$HOME/.gitignore`?

Reply:
Done.